demo/src/screens/componentScreens/MarqueeScreen.tsx

```diff
@@ -62,28 +61,32 @@ export default class MarqueeScreen extends Component<{}> {
   render() {
     return (
       <View flex padding-page>
-        <Text h1 center margin-20 $textDefault>
-          Marquee
-        </Text>
-        <View>
-          {renderMultipleSegmentOptions.call(this, 'Duration (optional)', 'duration', [
-            {label: '3000', value: 3000},
-            {label: '5000', value: 5000},
-            {label: '10000', value: 10000}
-          ])}
-          {renderMultipleSegmentOptions.call(this, 'Number Of Reps', 'numOfReps', [
-            {label: 'Infinite', value: -1},
-            {label: '1', value: 1},
-            {label: '3', value: 3},
-            {label: '5', value: 5}
-          ])}
-          <View marginV-s2>
-            {renderBooleanOption.call(this, 'Direction Horizontal: Left To Right/Right To Left', 'directionHorizontal')}
-            {renderBooleanOption.call(this, 'Direction Vertical:  Bottom To Up/Up To Bottom', 'directionVertical')}
+        <ScrollView>
```

Comment:
Why not replace the wrapping View with a ScrollView?

Reply:
You're right I'll just pass the style to the ScrollView